homeassistant/components/renault/config_flow.py

```diff
@@ -0,0 +1,93 @@
+"""Config flow to configure Renault component."""
+from __future__ import annotations
+
+from typing import Any
+
+from renault_api.const import AVAILABLE_LOCALES
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.data_entry_flow import FlowResult
+
+from .const import CONF_KAMEREON_ACCOUNT_ID, CONF_LOCALE, DOMAIN
+from .renault_hub import RenaultHub
+
+
+class RenaultFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a Renault config flow."""
+
+    VERSION = 1
+    CONNECTION_CLASS = config_entries.CONN_CLASS_CLOUD_POLL
```

Comment:
This has been moved to the manifest and is no longer needed here.
```suggestion
```

Reply:
Updated in 2487c993b53cb2b60bbda1c9eb5f115759c3b76e